doc-content/enterprise-only/openshift/template-overview-con.adoc

```diff
@@ -4,23 +4,23 @@
+* `rhpam712-prod-immutable-kieserver.yaml` provides an immutable {KIE_SERVER} instance. When you deploy this template, a source-to-image (S2I) build is triggered for one or several services that are to run on the {KIE_SERVER} instance. the {KIE_SERVER} instance can optionally be configured to connect to the {CENTRAL} Monitoring instance and Smart Router provided by `rhpam712-prod-immutable-monitor.yaml`. For details about this template, see <<rhpam712-prod-immutable-kieserver-ref_{context}>>.
+* `rhpam712-prod-immutable-kieserver-amq.yaml` provides an immutable {KIE_SERVER} instance. When you deploy this template, a source-to-image (S2I) build is triggered for one or several services that are to run on the {KIE_SERVER} instance. the {KIE_SERVER} instance can optionally be configured to connect to the {CENTRAL} Monitoring instance and Smart Router provided by `rhpam712-prod-immutable-monitor.yaml`. This version of the template includes JMS integration. For details about this template, see <<rhpam712-prod-immutable-kieserver-amq-ref_{context}>>.
+* `rhpam712-kieserver-externaldb.yaml` provides a {KIE_SERVER} instance that uses an external database. You can configure the {KIE_SERVER} instance to connect to a {CENTRAL} instance. Also, you can copy sections from this template into another template to configure a {KIE_SERVER} instance in the other template to use an external database. For details about this template, see <<rhpam712-kieserver-externaldb-ref_{context}>>.
+* `rhpam712-kieserver-mysql.yaml` provides a {KIE_SERVER} instance and a MySQL instance that the {KIE_SERVER} instance uses. You can configure the {KIE_SERVER} instance to connect to a {CENTRAL} instance. Also, you can copy sections from this template into another template to configure a {KIE_SERVER} instance in the other template to use MySQL and to provide the MySQL instance. For details about this template, see <<rhpam712-kieserver-mysql-ref_{context}>>.
+* `rhpam712-kieserver-postgresql.yaml` provides a {KIE_SERVER} instance and a PostgreSQL instance that the {KIE_SERVER} instance uses. You can configure the {KIE_SERVER} instance to connect to a {CENTRAL} instance. Also, you can copy sections from this template into another template to configure a {KIE_SERVER} instance in the other template to use PostgreSQL and to provide the PostgreSQL instance. For details about this template, see <<rhpam712-kieserver-mysql-ref_{context}>>.
+* `rhpam712-managed.yaml` provides a high-availability {CENTRAL} Monitoring instance, a {KIE_SERVER} instance, and a PostgreSQL instance that the {KIE_SERVER} instance uses. `OpenShiftStartupStrategy` is enabled, ensuring that the {CENTRAL} Monitoring instance can connect to other {KIE_SERVER} instances in the same project automatically, as long as these instances have OpenShiftStartupStrategy enabled as well.
```

Comment:
Please add:
For details about this template, see <<rhpam712-managed-ref_{context}>>.

Reply:
Added